receiver/prometheusreceiver/metrics_receiver_protobuf_test.go

```diff
@@ -257,12 +282,49 @@ func TestNativeVsClassicHistogramScrapeViaProtobuf(t *testing.T) {
 						{Offset: 2, Length: 1},
 					},
 					PositiveDelta: []int64{1, 0},
+					Exemplars: []*dto.Exemplar{
+						{
+							Value:     0.8,
+							Label:     []dto.LabelPair{{Name: "tnh_e1", Value: "mv1"}},
+							Timestamp: &types.Timestamp{Seconds: 234, Nanos: 456},
+						},
+						{
+							Value:     111,
+							Label:     []dto.LabelPair{{Name: "tnh_e2", Value: "mv2"}},
+							Timestamp: &types.Timestamp{Seconds: 432, Nanos: 456},
+						},
+					},
 				},
 			},
 		},
 	}
 	prometheusMetricFamilyToProtoBuf(t, buffer, nativeHistogram)
 
+	// Exemplar check function to check that the right values go to the
+	// right datapoints.
+	tchExemplarCheck := func(t *testing.T, hdp pmetric.HistogramDataPoint) {
```

Comment:
in general I prefer descriptive variable names instead of one letter ones. They are super hard to understand at first glance 😭

Reply:
done